src/styles/reset.css

```diff
@@ -0,0 +1,14 @@
+body,
+h2 {
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+h2 {
```

Comment:
I think it is better to have all the styles for h2 together
Because is it not only reset any more you could name the file `normalize.css`